infra/core/ai/formrecognizer.bicep

```diff
@@ -0,0 +1,26 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param customSubDomainName string = name
+param kind string = 'FormRecognizer'
+param publicNetworkAccess string = 'Enabled'
+param sku object = {
+  name: 'S0'
+}
+
+resource account 'Microsoft.CognitiveServices/accounts@2022-10-01' = {
+  name: name
+  location: location
+  tags: tags
+  kind: kind
+  properties: {
+    customSubDomainName: customSubDomainName
+    publicNetworkAccess: publicNetworkAccess
+  }
+  sku: sku
+}
+
+output endpoint string = account.properties.endpoint
+output id string = account.id
+output name string = account.name
```

Comment:
Instead of this, you can just reference the cognitiveservices.bicep module and pass it kind:FormRecognizer paramter in main.bicep.

If you want an explicit formrecognizer.bicep file, then you can also do the same thing.  That way we are reusing the cognitiveservices.bicep file and can have consistency there.

Reply:
Thanks for feedback. Addressed in PR: https://github.com/Azure-Samples/azure-search-openai-demo/pull/49